openapi-circe-yaml/src/main/scala/sttp/apispec/openapi/circe/yaml/SttpOpenAPICirceYaml.scala

```diff
@@ -4,10 +4,20 @@ import io.circe.syntax._
 import io.circe.yaml.Printer
 import io.circe.yaml.Printer.StringStyle
 import sttp.apispec.openapi.OpenAPI
-import sttp.apispec.openapi.circe._
 
 trait SttpOpenAPICirceYaml {
+
   implicit class RichOpenAPI(openAPI: OpenAPI) {
+    import sttp.apispec.openapi.circe._
+
+    def toYaml: String = Printer(dropNullKeys = true, preserveOrder = true).pretty(openAPI.asJson)
+    def toYaml(stringStyle: StringStyle): String =
+      Printer(dropNullKeys = true, preserveOrder = true, stringStyle = stringStyle).pretty(openAPI.asJson)
+  }
+
+  implicit class RichOpenAPI3_1(openAPI: OpenAPI) {
```

Comment:
I don't think we can add two implicit classes like this. `openAPI.toYaml` will be ambiguous if you `import sttp.apispec.openapi.circe.yaml._`

Reply:
Ok, do you think it would be ok to introduce methods like `openAPI.toYaml3_1` in the existing implicit class?

Reply:
yeah this might be a good option. Plus some scaladoc explaining what this means :)

Reply:
updated